### PR TITLE
Pulled in a few functions from LakeMetabolizer

### PR DESCRIPTION
**Pull request by aappling-usgs**
Addresses #5 halfway

### CODE REVIEW

**Comment by aappling-usgs** [comment]
AppVeyor build cancelled by user? What does that mean? Can I restart it?

**Comment by jread-usgs** [comment]
whoops. I cancelled the wrong one. restarting...

**Comment by aappling-usgs** [comment]
Oh, hang on. merging with upstream changes that fix the unitted dependency now.

File: R/o2.at.sat.R [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#' Calculates the equilibrium saturation concentration of oxygen in water at the supplied conditions
+#' 
+#' @importFrom LakeMetabolizer o2.at.sat
+o2.at.sat <- function(ts.data, baro, altitude=0, salinity=0, model='garcia'){
+  LakeMetabolizer::o2.at.sat(ts.data, baro, altitude, salinity, model)
```
Comment (jread-usgs) [inline R/o2.at.sat.R]: do we want to pull in o2.at.sat.base instead? Same question applies to the other ones that have a `ts.data` argument

Comment (aappling-usgs) [inline R/o2.at.sat.R]: x.base functions are fine with me.

**Pull request merged**